### DISCUSSION TITLE
MCMC output file is too large

### DISCUSSION THREAD

**Discussion by mamun-nahid** [AstroBridge/BDLensing#217] [Q&A]
MC output file is about 27mb when I use:

`with open(output_path, "wb") as f:
    joblib.dump([input, output], f, compress=True)`

It was reduced to 18mb by using  `compress=9`.

Is there any way to keep the file size below 1mb? 

notebook [link](https://colab.research.google.com/drive/1zX3SbgTcr3O-6GzZq0QhZF3liIQNL7rq?usp=sharing)

**Comment by mh-tanver** [comment]
Don't know whether it will work as you have already used  compress = 9 , Try **compress=('zlib', 9).**
